From 6e5c1db6b4a0af96afeb09a09fb2fc2b73308f01 Mon Sep 17 00:00:00 2001
From: Laszlo Ersek <lersek@redhat.com>
Date: Sun, 27 Aug 2023 09:19:09 +0200
Subject: [PATCH] tests/data: refresh Fedora tree URLs in virt-install-osinfo*
 expected XMLs

Libosinfo seems to generate Fedora tree URLs using the "https", not
"http", scheme now; which breaks CI. Update the expected outputs
accordingly.

Signed-off-by: Laszlo Ersek <lersek@redhat.com>
---
 .../cli/compare/virt-install-osinfo-unattended-treeapis.xml     | 2 +-
 tests/data/cli/compare/virt-install-osinfo-url-unattended.xml   | 2 +-
 tests/data/cli/compare/virt-install-osinfo-url-with-disk.xml    | 2 +-
 tests/data/cli/compare/virt-install-osinfo-url.xml              | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/data/cli/compare/virt-install-osinfo-unattended-treeapis.xml b/tests/data/cli/compare/virt-install-osinfo-unattended-treeapis.xml
index 1140bf0bb..5f3c3474e 100644
--- a/tests/data/cli/compare/virt-install-osinfo-unattended-treeapis.xml
+++ b/tests/data/cli/compare/virt-install-osinfo-unattended-treeapis.xml
@@ -13,7 +13,7 @@
     <type arch="x86_64" machine="pc-i440fx-6.1">hvm</type>
     <kernel>/VIRTINST-TESTSUITE/vmlinuz</kernel>
     <initrd>/VIRTINST-TESTSUITE/initrd.img</initrd>
-    <cmdline>ks=file:/fedora.ks method=http://archive.fedoraproject.org/pub/archive/fedora/linux/releases/17/Fedora/x86_64/os/</cmdline>
+    <cmdline>ks=file:/fedora.ks method=https://archive.fedoraproject.org/pub/archive/fedora/linux/releases/17/Fedora/x86_64/os/</cmdline>
   </os>
   <features>
     <acpi/>
diff --git a/tests/data/cli/compare/virt-install-osinfo-url-unattended.xml b/tests/data/cli/compare/virt-install-osinfo-url-unattended.xml
index fe653b8b4..2388120ac 100644
--- a/tests/data/cli/compare/virt-install-osinfo-url-unattended.xml
+++ b/tests/data/cli/compare/virt-install-osinfo-url-unattended.xml
@@ -13,7 +13,7 @@
     <type arch="x86_64" machine="q35">hvm</type>
     <kernel>/VIRTINST-TESTSUITE/vmlinuz</kernel>
     <initrd>/VIRTINST-TESTSUITE/initrd.img</initrd>
-    <cmdline>ks=file:/fedora.ks inst.repo=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/26/Server/x86_64/os/</cmdline>
+    <cmdline>ks=file:/fedora.ks inst.repo=https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/26/Server/x86_64/os/</cmdline>
   </os>
   <features>
     <acpi/>
diff --git a/tests/data/cli/compare/virt-install-osinfo-url-with-disk.xml b/tests/data/cli/compare/virt-install-osinfo-url-with-disk.xml
index 99fb90fb2..11fce0aad 100644
--- a/tests/data/cli/compare/virt-install-osinfo-url-with-disk.xml
+++ b/tests/data/cli/compare/virt-install-osinfo-url-with-disk.xml
@@ -13,7 +13,7 @@
     <type arch="x86_64" machine="q35">hvm</type>
     <kernel>/VIRTINST-TESTSUITE/vmlinuz</kernel>
     <initrd>/VIRTINST-TESTSUITE/initrd.img</initrd>
-    <cmdline>method=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/26/Server/x86_64/os/</cmdline>
+    <cmdline>method=https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/26/Server/x86_64/os/</cmdline>
   </os>
   <features>
     <acpi/>
diff --git a/tests/data/cli/compare/virt-install-osinfo-url.xml b/tests/data/cli/compare/virt-install-osinfo-url.xml
index 37fcc1091..ea1937a30 100644
--- a/tests/data/cli/compare/virt-install-osinfo-url.xml
+++ b/tests/data/cli/compare/virt-install-osinfo-url.xml
@@ -13,7 +13,7 @@
     <type arch="x86_64" machine="q35">hvm</type>
     <kernel>/VIRTINST-TESTSUITE/vmlinuz</kernel>
     <initrd>/VIRTINST-TESTSUITE/initrd.img</initrd>
-    <cmdline>method=http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/26/Server/x86_64/os/</cmdline>
+    <cmdline>method=https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/26/Server/x86_64/os/</cmdline>
   </os>
   <features>
     <acpi/>
